penguin/web/services/notification_settings.py

```
@@ -0,0 +1,79 @@
+    return value if value in allowed else default
+
+
+def _quiet_hours_from_env() -> dict[str, str] | None:
+    start = os.getenv("PENGUIN_TUI_NOTIFICATION_QUIET_START", "").strip()
+    end = os.getenv("PENGUIN_TUI_NOTIFICATION_QUIET_END", "").strip()
+    if not start or not end:
+        return None
+    return {"start": start, "end": end}
+
+
+def notification_settings_payload(core: Any | None = None) -> dict[str, Any]:
```

Ruff (0.15.20)

[warning] Unused function argument: `core`

(ARG001)